tests/gis_tests/gis_migrations/test_operations.py

```diff
@@ -202,6 +202,21 @@ def test_alter_geom_field_dim(self):
         )
         self.assertFalse(Neighborhood.objects.first().geom.hasz)
 
+    def test_oracle_field_arguments(self):
+        """
+        #29792 --- Oracle GIS field argument preservation in migrations.
+        """
```

Comment:
Remove ticket reference since the test is self documenting. The issue is not complicated enough to require extra context.

Also it feels like the patch should live in `gis_tests.test_fields` instead. Since `GeometryField.deconstruct` seems to be not directly tested it could worth adding full option coverage to the test. By that I mean testing `dim` and `geography` as well.